tests/drone/get-latest-ocis-commit-id.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+source .drone.env
+
+echo "$OCIS_BRANCH"
+# Run git ls-remote to get the latest commit ID
+latest_commit_id=$(git ls-remote https://github.com/owncloud/ocis.git "refs/heads/$OCIS_BRANCH" | cut -f 1)
+
+# Specify the path to the drone.env file
+env_file="./.drone.env"
+
+if [[ "$OCIS_BRANCH" == "master" ]] || [[ "$OCIS_BRANCH" == "stable-5.0" ]]; then
```

Comment:
in case if we want to use specific branch we should specify `OCIS_COMMITID`
and `OCIS_BRANCH` then the commit update will not happen, correct?

Reply:
This script runs only on nightly. On nightly, we fetch latest commit id of given ocis branch. On PR, we used specified commit id.

Reply:
Thanks, now I got it 👍